src/ExadelMentorship.WebApi/Controllers/MessageController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using ExadelMentorship.WebApi.RabbitMQ;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ExadelMentorship.WebApi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class MessageController : ControllerBase
+    {
+        private readonly IMessageProducer _messagePublisher;
+
+        public MessageController(IMessageProducer messagePublisher)
+        {
+            _messagePublisher = messagePublisher;
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> CreateOrder(string message)
```

Comment:
You can just return string itself! No need for IActionResult

Reply:
done